system-test/src/test/resources/TestFile2.java

```diff
@@ -0,0 +1,27 @@
+package org.digma.intellij.plugin.test.system
+        
+public class TestFile2 {
+
+    public TestFile2() {
+    }
+
+    
+    public void method1() {
+    }
+
+    public void method2() {
+    }
+
+    public void relatedSpan1() {
+    }
+
+    public void relatedSpan2() {
+    }
+
+    public void relatedEndpoint1() {
+    }
+
+    public void relatedEndpoint2() {
+    }
+
+}
```

Comment:
does the test also tests spans and endpoints?

Reply:
No, those are for the insights mock data. those methods id's are placed in the Insight as relatedSpan and Endpoint but they real location in the test files for us to navigate to them